lib/teiserver/mix_tasks/party_balance_stats.ex

```diff
@@ -0,0 +1,299 @@
+defmodule Mix.Tasks.Teiserver.PartyBalanceStats do
```

Comment:
We already have the results, will we need this again in future?

Reply:
Since I've made changes to the value where parties get broken, ideally we put this PR on integration again and rerun:
```
mix teiserver.party_balance_stats /var/log/teiserver/results.txt
```

Previously I hardcoded parties to be broken when rating diff > 14
Now I change to ignore parties when rating diff > max(10, 5% of team rating)